ethcore/src/machine.rs

```diff
@@ -122,7 +122,13 @@ impl EthereumMachine {
 }
 
 impl EthereumMachine {
-	/// Execute a call as the system address.
+	/// Execute a call as the system address. Block environment information passed to the
+	/// VM is modified to have its gas limit bounded at the upper limit of possible used
+	/// gases including this system call, capped at the maximum value able to be
+	/// represented by U256. This system call modifies the block state, but discards other
+	/// informations. If suicides, logs or refunds happened within the system call, they
```

Comment:
s/informations/information/
s/suicides, logs or refunds happened within the system call/suicides, logs or refunds happen within the system call/